judge/comments.py

```diff
@@ -118,5 +123,6 @@ def get_context_data(self, **kwargs):
             context['is_new_user'] = not self.request.user.is_staff and not profile.has_any_solves
         context['comment_list'] = queryset
         context['vote_hide_threshold'] = settings.DMOJ_COMMENT_VOTE_HIDE_THRESHOLD
+        context['reply_timeframe'] = timezone.now() - settings.DMOJ_COMMENT_REPLY_TIMEFRAME
```

Comment:
Let's call this something like `can_reply_before`.

Reply:
`{% if node.time > can_reply_before %}` is too confusing for me

Reply:
what about `reply_cutoff`

Reply:
sure